packages/tizen_rpc_port/example/example_proxy/lib/main.dart

```diff
@@ -0,0 +1,135 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'package:flutter/material.dart';
+
+import 'package:flutter/services.dart';
+import 'package:tizen_log/tizen_log.dart';
+
+import 'message_proxy.dart';
+
+// ignore_for_file: public_member_api_docs
+
+const String _logTag = 'TizenRpcPortProxyExample';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatefulWidget {
+  const MyApp({super.key});
+
+  @override
+  State<MyApp> createState() => _MyAppState();
+}
+
+String _message = '';
+
+class MyNotify extends NotifyCallback {
+  @override
+  Future<void> onReceived(String sender, String message) async {
+    Log.info(_logTag, 'onReceived $sender: $message');
+    _message = '$sender: $message';
+  }
+}
+
+class MyMessageProxy extends Message {
+  MyMessageProxy(super.appid);
+
+  String get message => _message;
+  set message(String message) => _message = message;
+
+  @override
+  Future<void> onConnected() async {
+    _message = 'onConnected';
+    Log.info(_logTag, 'onConnected');
+    print('onConnected');
+
+    await register('Native_GOGO1', MyNotify());
```

Comment:
I agree your suggestion.
Thanks for the review!

Reply:
Please also consider replacing other occurrences of

- proxy → client
- stub → server

in the example apps where appropriate (such as the directory names and application IDs) to avoid ambiguity.

Note: You shouldn't need the "example_" prefix in the directory names.

Reply:
Thanks for your suggestion.
I removed "example_" prefix, and changed `MyMessage~` class names.
However, the appid and directory names are for API examples, so I think it would be better not to modify them.Thanks for your suggestion.

Reply:
수정하신 내용에 무리는 없다고 생각하지만 먼저 용어들의 기본 의미를 짚고 넘어가겠습니다.

https://en.wikipedia.org/wiki/Remote_procedure_call

예를 들어 로컬에서 A라는 동작을 수행하는 프로그램이 있다고 하겠습니다. 만약 이 동작을 로컬이 아닌 원격 머신의 리소스를 활용하여 실행하게 된다면 요청을 하는 쪽은 client, 동작을 수행하고 결과를 반환하는 쪽은 server로 표현할 수 있습니다. 여기서 client는 기존과 유사한 코드를 사용하되 AProxy라는 객체를 통해 (일반적으로 client stub, server stub이라는 표현을 쓰지만 여기서는 기존에 사용하시던 용어에 맞춰 proxy, stub으로 표현하겠습니다) 명령을 실행하게 될 것이고 server는 그 반대에 해당하는 동작을 AStub을 통해 수행하게 될 것입니다. 여기서 proxy, stub은 marshalling과 unmarshalling을 수행하는 중간 매개체, 즉 단순한 코드 조각에 불과하며 client, server는 명령을 내리고 동작을 수행하는 "주체"에 해당할 것입니다.

제 코멘트는 일개 수단으로서의 역할을 하는 proxy, stub이라는 개념이 앱 이름 등 전면에 앞세워져 있어 의문을 표시했던 것입니다.

Reply:
client / server로 작성하는 것이 명확한 표현입니다.

@upple 님, @swift-kim 님 말씀처럼 수정 부탁드리겠습니다.

Reply:
안녕하세요. 의견 감사합니다.
말씀해주신 내용을 보고 생각해보니 client - server를 이름으로 사용하는게 더 적절하다고 생각되네요.
말씀처럼 proxy, stub는 통신을 위한 수단으로써 봐야 한다는 것에 동의합니다.
해당 의견 반영해서 패치 올렸습니다.

항상 좋은 리뷰 주셔서 감사합니다!